static/includes/clickhouse-storage-limitation.md

```diff
@@ -0,0 +1,9 @@
+:::note[Total storage per plan]
+In service plans with a single shard, storage capacity doesn't depend on the number of
+nodes, and the data of this single shard is replicated on all the available nodes.
+
+For example, with a Startup-16 plan you get 1150 GB of total storage per VM, same as with
```

Comment:
That is a good explanation.
I would also add a small clarification for the sharded plan here.

Reply:
Thank you @Khatskevich ! What would you specifically include for the sharded plan?

Reply:
May be give another example: For example we have Premium-6x-16 which has 2 shards and 6 servers.
This means that there are 6 servers with 1150GB storage each and the data is replicated 3 times.

And the conclusion: Total storage is the amount of data which may be inserted to the service, which will be replicated `number of nodes` / `number of shards` times.